src/metaswitch/clearwater/etcd_shared/plugin_utils.py

```diff
@@ -63,3 +63,16 @@ def run_command(command, namespace=None, log_error=True):
                                                          e.returncode,
                                                          e.output))
         return e.returncode
+
+
+def get_key():
```

Comment:
Should this be a function that reads /etc/clearwater/config, or a command-line parameter that gets passed through to the plugins?